### PR TITLE
feat(octavia): pool - add a member manually

### PR DESCRIPTION
**Pull request by karim-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/octavia-batch-1`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-10687
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add a member manually via the button in members list

### CODE REVIEW

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/template.html [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
-    <oui-header-tabs-item
+    <!--<oui-header-tabs-item
         data-href="{{:: $ctrl.healthMonitorPoolsDetailLink }}"
         data-active="$ctrl.currentActiveLink().startsWith($ctrl.healthMonitorPoolsDetailLink)"
     >
         <span
             data-translate="load_balancer_pools_detail_health_monitor_tab_title"
         ></span>
-    </oui-header-tabs-item>
+    </oui-header-tabs-item>-->
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/template.html:21]: No need to keep it in comment

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/service.js [hunk 2]
```diff
@@ -56,6 +56,21 @@ export default class OctaviaLoadBalancerPoolsService {
...
+          {
+            address,
+            name,
+            protocolPort,
+          },
+        ],
+      },
+    );
+  }
+
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/service.js:73]: Pu it in the member service

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    region: '<',
+    poolId: '<',
+    goBack: '<',
+    trackAction: '<',
+    trackPage: '<',
+    name: '<',
+    addressIp: '<',
+    port: '<',
+    trackCreateAction: '<',
+    trackCreatePage: '<',
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js:17]: This is not used

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    projectId: '<',
+    loadbalancer: '<',
+    region: '<',
+    poolId: '<',
+    goBack: '<',
+    trackAction: '<',
+    trackPage: '<',
+    name: '<',
+    addressIp: '<',
+    port: '<',
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js:15]: There is no need for these bindings

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+export default {
+  bindings: {
+    projectId: '<',
+    loadbalancer: '<',
+    region: '<',
+    poolId: '<',
+    goBack: '<',
+    trackAction: '<',
+    trackPage: '<',
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js:12]: These bindings are not used

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/module.js [hunk 6]
```diff
@@ -5,12 +5,13 @@ import routing from './routing';
 import service from './service';
 
 import list from './list';
+import create from './create';
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/module.js:8]: The module imports should be done in the octavia root module

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    Alerter,
+    OctaviaLoadBalancerPoolsService,
+    $translate,
+    $timeout,
+  ) {
+    this.$state = $state;
+    this.Alerter = Alerter;
+    this.OctaviaLoadBalancerPoolsService = OctaviaLoadBalancerPoolsService;
+    this.$translate = $translate;
+    this.timeout = $timeout;
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js:17]: This is not used

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  constructor(
+    $state,
+    Alerter,
+    OctaviaLoadBalancerPoolsService,
+    $translate,
+    $timeout,
+  ) {
+    this.$state = $state;
+    this.Alerter = Alerter;
+    this.OctaviaLoadBalancerPoolsService = OctaviaLoadBalancerPoolsService;
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js:15]: Use the members service

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+  /* @ngInject */
+
+  constructor(
+    $state,
+    Alerter,
+    OctaviaLoadBalancerPoolsService,
+    $translate,
+    $timeout,
+  ) {
+    this.$state = $state;
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js:13]: This is not used

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
+import controller from './controller';
+import template from './template.html';
+
+export default {
+  bindings: {
+    projectId: '<',
+    loadbalancer: '<',
```
Comment (JacquesLarique) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js:7]: I don't think you need the loadbalancer here

**Review by JacquesLarique — CHANGES_REQUESTED** [review]

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/template.html (hunk 1 above)
Comment (karim-ovh) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/template.html:21]: OK. I removed it and cleaned the routing file as well.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/service.js (hunk 2 above)
Comment (karim-ovh) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/service.js:73]: OK done.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js (hunk 3 above)
Comment (karim-ovh) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/component.js:17]: trackCreateAction is used but not trackCreatePage.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+          null,
+          'octavia.alerts.members',
+        );
+        this.goBack(true);
+      })
+      .catch((error) => {
+        this.Alerter.error(
+          this.$translate.instant('octavia_load_balancer_global_error', {
+            message: error.data?.message,
+            requestId: error.config?.headers['X-OVH-MANAGER-REQUEST-ID'],
```
Comment (qpavy) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/create/controller.js:39]:
```suggestion
            requestId: error.headers('X-Ovh-Queryid'),
```

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**